.ci_support/linux64.yaml

```diff
@@ -4,7 +4,11 @@ cxx_compiler:
   - gxx
 fortran_compiler:
   - gfortran
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- '10.1'
 channel_sources:
   - conda-forge,defaults
 docker_image:
-  - condaforge/linux-anvil-comp7
+  - condaforge/linux-anvil-cuda:10.1
```

Comment:
Ditto for these. 

cc @jakirkham for advice on the cuda bits and staged recipes

Reply:
Removed these – will the use of the cuda compiler activate these images when this becomes a feedstock?

Reply:
I hope? I am not familiar with the cuda stuff